examples/language-modeling/eval/evaluation.py

```diff
@@ -44,41 +45,42 @@
     "winogrande": [0, 5],
     "piqa": [0],
     "coqa": [],  ## coqa is not enabled in llamav1 models
-    "truthfulqa_mc": [0],
+    "truthfulqa_mc1": [0],
+    "truthfulqa_mc2": [0],
     "openbookqa": [0],
     "boolq": [0],
     "rte": [0],
     "arc_easy": [0, 25],
     "arc_challenge": [0, 25],
-    "hendrycksTest-*": [0, 5],
+    "mmlu": [0, 5],
     "wikitext": [0],
     "drop": [3],
     "gsm8k": [5]
 }
 
 
 def simple_evaluate(
-        model,
-        model_args=None,
-        tasks=[],
-        num_fewshot=0,
-        batch_size=None,
-        max_batch_size=None,
-        device=None,
-        no_cache=True,
-        limit=None,
-        bootstrap_iters=100000,
-        description_dict=None,
-        check_integrity=False,
-        decontamination_ngrams_path=None,
-        write_out=False,
-        output_base_path=None,
-        lm=None
+    model,
+    model_args=None,
+    tasks=[],
+    num_fewshot=None,
```

Comment:
better keep the old version for a while to help users reproduce the result in our paper/blog